timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchEventIntegrationTest.java

```diff
@@ -79,6 +81,10 @@ public final class LockWatchEventIntegrationTest {
     private static final byte[] DATA_2 = "as".getBytes(StandardCharsets.UTF_8);
     private static final byte[] DATA_3 = "usual".getBytes(StandardCharsets.UTF_8);
     private static final byte[] DATA_4 = "I see".getBytes(StandardCharsets.UTF_8);
+    private static final ChangeMetadata CHANGE_METADATA_1 = ChangeMetadata.created(DATA_1);
+    private static final ChangeMetadata CHANGE_METADATA_2 = ChangeMetadata.deleted(DATA_2);
+    private static final ChangeMetadata CHANGE_METADATA_3 = ChangeMetadata.unchanged();
```

Comment:
nit: maybe just call these what they're respectively? I.e. `DELETE_CHANGE_METADATA`, etc

Reply:
^ re-ping on this